src/client/input/sdl.c

```diff
@@ -2322,7 +2330,7 @@ IN_Init(void)
 		gyro_active = true;
 	}
 
-	windowed_pauseonfocuslost = Cvar_Get("windowed_pauseonfocuslost", "0", CVAR_USERINFO | CVAR_ARCHIVE);
+	windowed_pauseonfocuslost = Cvar_Get("vid_pauseonfocuslost", "0", CVAR_USERINFO | CVAR_ARCHIVE);
```

Comment:
`vid_pauseonfocuslost` is still missing documentation in the General section in https://github.com/yquake2/yquake2/blob/master/doc/040_cvarlist.md